pkgs/development/lua-modules/overrides.nix

```diff
@@ -273,9 +276,21 @@ with super;
      sed -i 's,\(option(WITH_SHARED_LIBUV.*\)OFF,\1ON,' CMakeLists.txt
      rm -rf deps/libuv
     '';
+  });
+
+  luv-dev = super.luv.override({
     propagatedBuildInputs = [
       pkgs.libuv
     ];
+    preBuild = ''
+     sed -i 's,\(option(WITH_SHARED_LIBUV.*\)OFF,\1ON,' CMakeLists.txt
+     sed -i 's,\(option(BUILD_MODULE.*\)ON,\1OFF,' CMakeLists.txt
+     sed -i 's,$''+''{INSTALL_INC_DIR},${placeholder "out"}/include/luv,' CMakeLists.txt
```

Comment:
can't we use substituteAll instead ? better use nix abstraction tools.

Reply:
`sed` seems more consistent with the rest of the file